lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -117,14 +118,19 @@ async function runSmokeTestDefn(concurrentMapper, smokeTestDefn, defnOptions) {
   const {id, config: configJson, expectations} = smokeTestDefn;
   const {concurrency, lighthouseRunner, retries, isDebug} = defnOptions;
 
-  const individualTests = expectations.map(expectation => ({
-    requestedUrl: expectation.lhr.requestedUrl,
-    configJson,
-    expectation,
-    lighthouseRunner,
-    retries,
-    isDebug,
-  }));
+  const individualTests = expectations.map(expectation => {
+    if (expectation.networkRequests && concurrency > 1) {
```

Comment:
I think we still want to be able to run in parallel locally, at least. Maybe move this check to `pruneExpectations()` in `report-assert.js` and remove `expectation.networkRequests` if `!process.env.CI && concurrency > 1` (to enforce doing the extra checks in CI), otherwise throw this error?

Reply:
Is there a way to know `concurrency` in  `pruneExpectations`?

Reply:
no, it would have to be passed down in `reportOptions` (either directly or as a `shouldTestNetworkRequests` or whatever)